### DISCUSSION TITLE
Feedback/log after starting an upload ?

### DISCUSSION THREAD

**Discussion by buha** [simulot/immich-go#203] [Q&A] [answered]
So yesterday I used immich-go to upload a bunch of files recursively. For whatever reason I stopped it at some point and today I ran the same command on the same directory to continue the upload process.

I expect there will be a lot of duplicates which immich-go will detect and not upload.

Now, it's been sitting for hours like this:
```
$ immich-go -server=http://192.168.0.3:2283 -key=[my key] upload library/
immich-go  0.13.0, commit ae3095aa33fd7b38e418b07bc7e10aab6e0d800f, built at 2024-03-24T15:56:41Z
Server status: OK
Ask for server's assets...
14601 asset(s) received
Browsing folder(s)...
Done.
```

And I have no way to tell where it's at, how long it will take and so on. Is there a log file somewhere ? Or any way I could tell where it's at ?

**Comment by simulot** [comment, marked as answer]
I'm working on displaying progress bars... You'll have a visual feedback.

You can increase the log level with this option:
`-log-level=INFO`

And save the log in a file:
`-log-file=file`
